src/lua/util.rs

```diff
@@ -317,3 +321,50 @@ pub fn to_yaml<'a>(util: Table<'a>, lua: &Lua) -> Result<Table<'a>> {
     util.set("to_yaml", func)?;
     Ok(util)
 }
+
+/// Get a style object for the given path
+///
+/// Type: function( path ) -> style
+///
+/// Example:
+///
+/// ```lua
+/// xplr.util.lscolor("Desktop")
+/// -- { fg = "Red", bg = nil, add_modifiers = {}, sub_modifiers = {} }
+/// ```
+pub fn lscolor<'a>(util: Table<'a>, lua: &Lua) -> Result<Table<'a>> {
+    let lscolors = LsColors::from_env().unwrap_or_default();
+    let func = lua.create_function(move |lua, path: String| {
+        let style = lscolors
+            .style_for_path(path)
+            .map_or(Style::default(), Style::from);
+        lua::serialize(lua, &style).map_err(LuaError::custom)
+    })?;
+    util.set("lscolor", func)?;
+    Ok(util)
+}
```

Comment:
Since `style` is capable of handling `nil` as input, I'd expect it to return `nil` instead of `Style::default()` to avoid unnecessary serialization.